src/components/Map/Map.tsx

```diff
@@ -10,14 +12,57 @@ interface IProps{
 }
 
 const Map: React.FC<IProps> = ( {countries} ) => {
+  const [selectedCountry, setSelectedCountry]= useState<any>('Select A Country')
+  
+  const [displayCountry, setDisplayCountry] = useState<boolean>(false)
+  const countryNames = countries.map(country => { 
+    return <option key={uuidv4()}>{country.name}</option>
+  })
+
+   // ********* Function to get chosen country for QuizPage component ********//
+   const getCurrentCountry = () => {
```

Comment:
We can decide if we want to keep this or do a fetch for a single country. Right now, it seems like this option is less 'expensive' and would avoid any possible server issues. I'm curious to hear both of your thoughts!